intuita-webview/src/sourceControl/CreateIssue/index.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+	VSCodeButton,
+	VSCodeTextArea,
+	VSCodeTextField,
+} from '@vscode/webview-ui-toolkit/react';
+import { useEffect, useState } from 'react';
+import { vscode } from '../../shared/utilities/vscode';
+import styles from './style.module.css';
+import { IssueFormData } from '../../../../src/components/webview/webviewEvents';
+
+type Props = Readonly<{
+	title: string;
+	body: string;
+}>;
+
+const CreateIssue = ({ title, body }: Props) => {
+	const [loading] = useState(false);
```

Comment:
why we dont have state setter here?

Reply:
b/c this functionality is not yet connected to the backend